docs: account for audit mode and skipping comments (#36)
phylum-dev/phylum-analyze-pr-action: README.md

Author: This comment is meant to serve as a reminder to NOT merge this PR until https://github.com/phylum-dev/phylum-ci/pull/400 is complete, merged, and part of a `phylum-ci` release.

Author: `phylum-ci` v0.43.0 was released with the changes.

          # Consider using a bot or group account for this token.
          phylum_token: ${{ secrets.PHYLUM_TOKEN }}

>           # NOTE: These are examples. Only one `github_token` entry line is expected,
>           #       and only with comment generation enabled.

Reviewer: Even when comment generation is enabled, the default can be used. So I think this wording is misleading.

```suggestion
          # NOTE: These are examples. At most one `github_token` entry line is needed.
```
